lib/view.js

```diff
@@ -0,0 +1,22 @@
+// Native
+const path = require('path')
+const fs = require('fs')
+const { promisify } = require('util')
+
+// Packages
+const { compile } = require('handlebars')
+
+const readFileAsync = promisify(fs.readFile)
+
+module.exports = async () => {
+  let viewContent = false
+  const viewPath = path.normalize(path.join(__dirname, '/../views/index.hbs'))
+
+  try {
+    viewContent = await readFileAsync(viewPath, 'utf8')
+  } catch (err) {
+    throw err
```

Comment:
If you don't handle the error, you can just write:

```js
const viewContent = await readFileAsync(viewPath, 'utf8')
```

Without the try/catch block 😊

Reply:
This also means you can remove the `let viewContent = false` at the top... 🚀